torch/nn/modules/pooling.py

```diff
@@ -568,6 +568,15 @@ class AvgPool2d(_AvgPoolNd):
         >>> output = m(input)
     """
 
+    def __init__(self, kernel_size, stride=None, padding=0, ceil_mode=False,
+                 count_include_pad=True):
+        super(AvgPool2d, self).__init__()
+        self.kernel_size = kernel_size
```

Comment:
This probably needs a `_pair`

Reply:
It's used to be without `_pair` before 605307f.

Reply:
i presume it _pair is done in C or something....